### DISCUSSION TITLE
CSS Mixins nativos com seletor global e custom properties valem a pena?

### DISCUSSION THREAD

**Discussion by afonsopacifer** [frontendbr/forum#417] [Discussão]
Oie.. 💜 
Vi a Lea Verou (como sempre) fazendo coisas absurdamente incríveis outro dia..
Uma me chamou atenção, uma forma de criar **mixins utilizando variáveis**, gostaria de propor uma discussão e saber **o que vocês acham disso??**... ps: Eu gosto muuuuito haha

Para conseguir discutir melhor sobre o assunto, eu fiz um **video explicando como isso funciona**:
Mixins Nativos: https://youtu.be/hs1mASz6coA

PS: Para quem não manja de Custom Properties (variáveis) eu fiz dois outros **videos introdutórios** tbm, só para nivelar os pensamentos sobre o assunto:

Basico: https://youtu.be/NAp1FX9mGPU
Fallbacks: https://youtu.be/3joOXJkSM58 

[![003](https://cloud.githubusercontent.com/assets/6748866/22249962/4bfcbe20-e22c-11e6-9bd6-7f7bbea78a3a.jpg)](https://youtu.be/hs1mASz6coA)

XOXO

PS: Antes que falem sobre pré-processadores, fique claro que isso é nativo e roda em runtime, logo vc pode alterar um valor desses com @media ou javascript EM PRODUÇÃO e fazer coisas muuito incríveis.. esse é o ponto quando se discute sobre Custom Properties, tentem não perder tempo com comparações a SASS ou Stylus, pq NÃO FAZ SENTIDO ❤️

**Comment by Lucasktrindade** [comment]
Você ganha essa vantagem de pode mudar a variável em runtime e vai conseguir reduzir e reaproveitar o código muito mais, o que já diferencia os mixins de pré processadores que são usados mais para helpers.
